src/redux/account/reducer.js

```diff
@@ -0,0 +1,43 @@
+import {
+  LOGIN_USER_START,
+  LOGIN_USER_SUCCESS,
+  LOGIN_USER_FAILURE,
+  LOGOUT_SUCCESS,
+} from './actionType';
+
+const initialState = {
+  accessToken: '',
+  error: '',
+  isLoggingIn: false,
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case LOGIN_USER_START:
+      return {
+        ...state,
+        isLoggingIn: true,
+      };
+    case LOGIN_USER_SUCCESS:
+      return {
+        ...state,
+        accessToken: action.payload,
+        isLogginIn: false,
```

Comment:
did you try your code? This shouldn't work
```suggestion
        isLoggingIn: false,
```